helper_dart/lib/src/hits_searcher.dart

```diff
@@ -226,6 +226,7 @@ class _HitsSearcher with DisposableMixin implements HitsSearcher {
       ..add(
         _responses.listen((value) {
           eventTracker.trackViews(
+            snapshot().indexName,
```

Comment:
named arguments may be nicer for trackViews

Reply:
Good point! Fixed ✅